test/src/cli/flutter_cli_test.dart

```diff
@@ -462,7 +462,7 @@ void main() {
         expect(
           stdoutLogs,
           equals([
-            'Running "flutter test" in ${p.dirname(tempDirectory.path)}...\n',
+            'Running "flutter test" in .${p.context.separator}....\n',
```

Comment:
I think reading `./.`  as path is not ideal. I think we should avoid prefixing `./` and simply report `.` in this scenario.

Reply:
Yeah, I can see that! `.` is a bit easier on the eyes than `./.`